src/snmalloc/backend_helpers/globalrange.h

```diff
@@ -11,28 +11,15 @@ namespace snmalloc
   template<typename ParentRange>
   class GlobalRange
   {
-    typename ParentRange::State parent{};
+    static inline ParentRange parent{};
 
     /**
      * This is infrequently used code, a spin lock simplifies the code
      * considerably, and should never be on the fast path.
      */
-    FlagWord spin_lock{};
+    static inline FlagWord spin_lock{};
 
   public:
-    class State
-    {
-      SNMALLOC_REQUIRE_CONSTINIT static inline GlobalRange global_range{};
```

Comment:
This drops the `SNMALLOC_REQUIRE_CONSTINIT` here; should we put it on the `GlobalMetaRange global_state` object in `backend.h`?

Reply:
Should probably be on line 14 and 20 of this file.

Reply:
Agreed, fixed.